src/data/plugins.json

```
@@ -738,6 +738,13 @@
           "link": "https://github.com/MohamadKh75/cypress-plugin-dotenv",
           "keywords": ["testing", "commands", ".env", "environment variables"],
           "badge": "community"
+        },
+        {
+          "name": "cypress-intercept-formdata",
+          "description": "Work with Cypress' Intrecept multipart/form-data requests",
```

```suggestion
          "description": "Work with Cypress' Intercept multipart/form-data requests",
```

The same typo is also on the website description https://github.com/yoavniran/cypress-intercept-formdata in the About section.

---

thanks! will fix in repo too